pkg/tests/utils.go

```diff
@@ -33,9 +33,10 @@ var (
 	testCPU    = float64(0.99)
 	testMemory = float64(1.5)
 	port       = int32(80)
+	gpuSKUP100 = azaci.GpuSKUP100
```

Comment:
You don't need to copy this.

Reply:
Same. I had to as the `SKU` is `*string`, but the `GpuSKUP100` is not, so this is for solving the indirect pointer access.